src/com/amazon/ion/IonWriter.java

```diff
@@ -261,6 +261,17 @@
      */
     public void addTypeAnnotation(String annotation);
 
+    /**
+     * Adds a given symbolToken to the list of pending annotations.
+     * <p>
+     * The list of pending annotations is cleared when the current value is
+     * written via {@link #stepIn(IonType) stepIn()} or one of the
+     * {@code write*()} methods.
+     *
+     * @param annotation string annotation to append to the annotation list
+     */
+    public void addTypeAnnotationSymbol(SymbolToken annotation);
```

Comment:
Inheritance, https://github.com/amzn/ion-java/blob/masterAppendMerge/src/com/amazon/ion/impl/_Private_IonWriter.java#L26

Reply:
An lst and a user writer share functionality.

Reply:
I believe its because of the inheritance here as well, This is with public removed:
/Users/wesboyt/Dev/ion-java/src/com/amazon/ion/impl/bin/IonManagedBinaryWriter.java:[192,22] cannot find symbol
  symbol:   method addTypeAnnotationSymbol(com.amazon.ion.SymbolToken)
  location: variable currentWriter of type com.amazon.ion.impl._Private_IonWriter